File: .sdk/tm/php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// TempMailApiByBoomlify SDK utility: prepare_body

class TempMailApiByBoomlifyPrepareBody
{
    public static function call(TempMailApiByBoomlifyContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}

File: go/utility/prepare_body.go
package utility

import "github.com/voxgig-sdk/temp-mail-api-by-boomlify-sdk/go/core"

func prepareBodyUtil(ctx *core.Context) any {
	op := ctx.Op

	if op.Input == "data" {
		body := ctx.Utility.TransformRequest(ctx)
		return body
	}

	return nil
}
